src/cdk/schematics/ng-update/test-cases/misc/external-resource-resolution_input.ts

```diff
@@ -1,19 +0,0 @@
-import {Component} from '@angular/core';
-
-@Component({
-  selector: 'test-cmp',
-  templateUrl: '/projects/material/test.html',
-})
-export class MyTestComp {}
-
-@Component({
-  selector: 'test-cmp2',
-  templateUrl: '../some-tmpl.html',
-})
-export class MyTestComp2 {}
-
-@Component({
-  selector: 'test-cmp3',
-  templateUrl: 'local.html',
-})
-export class MyTestComp3 {}
```

Comment:
Actually, @alan-agius4 why have these been removed? These were general misc test assertions ensuring we support external templates, same for the other misc tests?

Reply:
I'm good removing these as the whole test case setup is not super convenient, but thought I would check

Reply:
These have been removed because they cannot run, as the migration was for version 6.

https://github.com/angular/components/blob/94b694f6cd54137c42acc5a2ded26353b18e3e71/src/cdk/schematics/ng-update/test-cases/misc/external-resource-resolution.spec.ts#L8

Reply:
They were just using *any* migration version to test the file being detected by the update tool.

It could totally be set to something else too

Reply:
I see, if you think we should keep it. I’ll try to update the spec to use the v15 migration.

Reply:
@devversion, I had a look at this tests and I am a bit confused by the above comment. As these migrations do not run for version 15 as these are always disabled.

https://github.com/angular/components/blob/68d81a48c6b01d796261c48b232f899c777e09fb/src/cdk/schematics/ng-update/migrations/method-call-arguments.ts#L24

https://github.com/angular/components/blob/68d81a48c6b01d796261c48b232f899c777e09fb/src/cdk/schematics/ng-update/data/method-call-checks.ts#L18

Potential post this PR, there needs to another PR to further clean up these files.

Reply:
Sorry, these misc tests just use any arbitrary target version, as long as there is at least one code transformation. We assert that the external template is then picked up and migrated properly (with the goal of testing that external files are recognized by our migration tooling - `external-resource-resolution.spec.ts`)

I'm good not re-adding these since it's definitely cumbersome if they rely on version-specific migration rules. It will be a pain whenever we delete redundant migrations. A loss for coverage but easier to maintain in the future.